packages/browsers/src/fileUtil.ts

```diff
@@ -89,3 +89,21 @@ async function installDMG(dmgPath: string, folderPath: string): Promise<void> {
     await exec(`hdiutil detach "${mountPath}" -quiet`);
   }
 }
+
+export async function configureWindowsSandboxPermissionsForChrome(
```

Comment:
This seems out of place in this file, it does not have relation to files.

Reply:
it sets file permissions